apps/verticalhome/js/app.js

```diff
@@ -126,10 +132,19 @@
             window.dispatchEvent(new CustomEvent('moz-app-loaded'));
           });
       }.bind(this));
+
+      this.pinManager = new PinAppManager();
+      navigator.mozL10n.ready(this.l10nInit.bind(this));
+      window.addEventListener('pin-app-loaded', function(e) {
+        this.pinManager.init();
+
+        this.pinNavigation = new PinAppNavigation(document.getElementById('pin-apps-list'));
+        this.pinNavigation.points_selector = '#pin-apps-list .pin-app-item';
+      }.bind(this));
```

Comment:
I don't get what any of this code is doing. It is my first time reading through it, but this seems like a place where comments would be helpful.